servers/src/common/stats.rs

```diff
@@ -51,7 +52,7 @@ pub struct ServerStats {
 	/// Number of peers
 	pub peer_count: u32,
 	/// Chain head
-	pub head: chain::Tip,
+	pub chain_stats: ChainStats,
```

Comment:
Want to replace `header_head` with a `ChainStats` as well? 
It would be useful to see the timestamp for the header chain during the initial header sync.

Reply:
yep will do